exporter/azuredataexplorerexporter/README.md

```diff
@@ -0,0 +1,163 @@
+# Azure Data Explorer Exporter
+
+This exporter sends metrics, logs and trace data to [Azure Data Explorer](https://azure.microsoft.com/services/data-explorer/).
+
+## Configuration
+
+The following settings are required:
+
+- `cluster_uri` (no default): The cluster name of the provisioned ADX cluster to ingest the data.
+- `application_id` (no default): The client id to connect to the cluster and ingest data.
+- `application_key` (no default): The cluster secret corresponding to the client id.
+- `tenant_id` (no default): The tenant id where the application_id is referenced from.
+
+The following settings can be optionally configured and have default values:
+> Note that the database , tables are expected to be created upfront before the exporter is in operation , the definition of these are in the section [Database and Table definition scripts](#database-and-table-definition-scripts)
+- `db_name` (default = "oteldb"): The ADX database where the tables are present to ingest the data.
```

Comment:
This is just used for default lookup. We have added the statement mentioning the DB and table requirement:
`Note that the database, tables are expected to be created upfront before the exporter is in operation, the definition of these are in the section [Database and Table definition scripts]`

Since we cannot leave the DB and table name empty, we have mentioned that the default names that ADX Exporter will be looking for.